docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -122,6 +122,31 @@ You'll need to replace these values in the policy example below:
 }
 ```
 
+## Step 4. TLS certificates for Teleport
+
+We now need to configure TLS certificates for Teleport to secure its communications.
+
+There are two options when using AWS:
+- Use `cert-manager` to provision and automatically renew ACME certs (described in [Step 4a](#step-4a-install-and-configure-cert-manager-to-handle-tls))
+  - This is better supported overall and will enable all Teleport services such as PostgreSQL database access
+    and CLI application access to work correctly. This is the recommended deployment option.
+- Use [AWS ACM](https://aws.amazon.com/certificate-manager/) to handle TLS termination with AWS-managed certificates (described in [Step 4b](#step-4b-use-acm-to-handle-tls))
+  - The use of ACM will not allow Teleport to handle PostgreSQL database connections or application access via CLI, as
+    Teleport will not be handling its own TLS termination.
+
+You must choose only one of these options.
+
+## Step 4a. Install and configure cert-manager to handle TLS
+
+Reference: [`cert-manager` docs](https://cert-manager.io/docs/)
```

Comment:
I think we should move this to the "Next steps" section or remove it. Currently, this link breaks the flow of the guide, since readers need to browse the cert-manager docs before they know how those docs are relevant for this guide.